CONTRIBUTING.md

```diff
@@ -189,6 +189,46 @@ Take a look at the sections that follow to pick a route (or five) that appeal to
 
 If you ever find yourself at a loss for what to do, or in need of mentorship or advice on how to contribute to Bevy, feel free to ask in [Discord] and one of our more experienced community members will be happy to help.
 
+### Join a working group
+
+Active initiatives in Bevy are organized into temporary working groups: choosing one of those and asking how to help can be a fantastic way to get up to speed and be immediately useful.
+
+Working groups are public, open-membership groups that work together to tackle a broad-but-scoped initiative.
+The work that they do is coordinated in a forum-channel on [Discord](https://discord.gg/bevy), although they also create issues and may use project boards for tangible work that needs to be done.
+
+There are no special requirements to be a member, and no formal membership list.
+Working groups are *spaces*, not clubs.
+
+### Start a working group
+
+When tackling a complex initiative, friends and allies can make things go much more smoothly.
+
+To start a working group:
+
+1. Decide what the working group is going to focus on. This should be tightly focused and achievable!
+2. Gather at least 3 people including yourself who are willing to be in the working group.
+3. Ping the `@Maintainer` role on Discord in #engine-dev announcing your mutual intent and a one or two sentence description of your plans.
+
+The maintainers will briefly evaluate the proposal in consultation with the relevant SMEs and give you a thumbs up or down on whether this is something Bevy can and wants to explore right now.
+You don't need a concrete plan at this stage, just a sensible argument for both "why is this something that could be useful to Bevy" and "why there aren't any serious barriers in implementing this in the near future".
+If they're in favor, a maintainer will create a forum channel for you and you're off to the races.
+
+Your initial task is writing up a design doc: laying out the scope of work and general implementation strategy.
+Once that's ready, get a sign-off on the broad vision and goals from the appropriate SMEs and maintainers.
+This is the primary review step: maintainers and SMEs should be broadly patient and supportive even if they're skeptical until a proper design doc is in hand to evaluate.
+
+With a sign-off in hand, post the design doc to [Github Discussions](https://github.com/bevyengine/bevy/discussions) for archival purposes and begin work on implementation.
```

Comment:
Perhaps a category should be created in the Github Discussions specifically for working group design docs? This will make it easier to find them in the future.

Reply:
This is a good idea. I don't think we will end up with too many of them.